Ginger/Ginger/ApplicationModelsLib/POMModels/PomAllElementsPage.xaml.cs

```diff
@@ -33,22 +52,22 @@ public enum eElementsContext
 
         }
 
+        public static bool DriverIsBusy { get; set; }
+
```

Comment:
Why DriverIsBusy is static? PomAllElementsPage can have instanses on multiple POM's + on Wizard